templates/docs/settings/table-layout.md

```diff
@@ -8,7 +8,17 @@ By default, tables in Vanilla use `table-layout: fixed`.
 
 There are cases where you might want to use `table-layout: auto` - e.g. in automatically generated tables. To do this, you need to set the `$table-layout-fixed` variable to `true`.
 
-Depending on the value of `$table-layout-fixed`, a utility class is generated to allow overriding if necessary:
+Two utility classes are generated to allow overriding if necessary:
 
-- If `$table-layout-fixed` is set to `true` (the default), you can use `u-table-layout--auto` to override it.
-- if `$table-layout-fixed` is set to `auto` it adds a utility called `u-table-layout--fixed`.
+- `u-table-layout--auto` sets a table and its descendant tables to `table-layout: auto`.
```

Comment:
We don't want people to nest tables, so "its descendant tables" may be a bit misleading, so we should say that this utility sets "the table or any descendant tables of the element it's put on" to auto layout.

Reply:
Updated documentation language accordingly